Tutorials/whatiswot/website/docs/Payloads and Headers.md

```diff
@@ -5,37 +5,51 @@ id: payloads
+
+![payloads and headers](/img/tutorial/Payloads-And-Headers/payloadsandheaders.png)
+
+## What is a payload?
 
-In computing, the payload is the part of transmitted data that is the actual intended message of the application and nothing to do with the protocol itself.
-It is also called the body of the message. It is not mandatory in most protocols.
+In computing, the payload is the part of transmitted data that is the actual intended message of the application and has nothing to do with the protocol itself. It is also called the body of the message. It is not mandatory in most protocols.
 
-We can approach with an analogy, where we consider protocols as a truck. A payload should not tell what it should be done by itself. In that sense, we can think of protocol as a truck.
-Here, we can think of the parcel contents as the payloads of a protocol.
-The parcel contents should not contain any information about their destination, the fact that they are transported with a truck or the color of the truck.
-In a protocol, payloads are very similar to parcel contents carried by a truck
+> A payload should not tell what should be done by itself. We can better explain this concept with an analogy, where we consider the protocol as a truck. We can think of the payload contents as the goods that this truck is transporting. The truck's goods should not contain any information about their destination, the fact that they are transported with a truck or the color of the truck.
+
+## What is a header?
 
 The header is part of a message that can and should be understood by the protocol stack. All the information that the payload should not contain, goes to the header.
 
-Let's continue with the parcel-truck example. In that example, the header would be the etiquette on the parcel.
+> Let's go back to our truck example. In that example, the header would be the label on the goods - indicating where they are being transported, what they are and etc.
+
+![truck with parcels example](/img/tutorial/Payloads-And-Headers/truck.png)
+
+## Header Example
 
 This is what an HTTP header can look like. A header can contain information like request context, response context, status and it can indicate payload type.
 
-There are different payload types: JSON, XML, CBOR, Text, Audio…
+![http header example](/img/tutorial/Payloads-And-Headers/httpheader.png)
 
-So let us show you an example of JSON payload. This is a temperature payload, a single number. And this is a configuration payload. It is a JSON object consisting of key/ value pairs desiredUnit and updateInterval. The temperature payload is sent from the sensor and the configuration is sent to the sensor.
+## Payload Types
+
+There are different payload types: JSON, XML, CBOR, text and audio to name a few.
+
+So let us show you an example of a **JSON payload**. 
```

Comment:
In cases where we show code or json, we should not put an image but simply keep the code with correct highlighting